packages/calcite-components/src/utils/focusTrapComponent.ts

```diff
@@ -58,8 +58,6 @@ export function connectFocusTrap(component: FocusTrapComponent, options?: Connec
   }
 
   const focusTrapOptions: FocusTrapOptions = {
-    clickOutsideDeactivates: true,
```

Comment:
Can we confirm falling back on the defaults is desired (e.g., `clickOutsideDeactivates: false`, `escapeDeactivates: true`)?

Reply:
I think we do want escape to deactivate by default on all components if they are focus trap components, and `clickOutsideDeactivates` is false on all except `popover`: under certain conditions. Added a comment for `sheet` as well to make it clear. 

focusTrapOptions: {
        // Scrim has it's own close handler, allow it to take over.
        clickOutsideDeactivates: false,